mpyl-example.py

```diff
@@ -9,7 +9,7 @@ def main(log: Logger, args: argparse.Namespace):
         from src.mpyl.reporting.targets.jira import JiraReporter
         from src.mpyl.steps.models import RunProperties
         from src.mpyl.utilities.pyaml_env import parse_config
-        from src.mpyl.cli.commands.build.mpyl import (
+        from src.mpyl.mpyl import (
```

Comment:
I like the idea of pulling these namespaces up a bit. But I don't really like the `mpyl.mpyl` namespace. What's wrong with `cli`? Isn't that descriptive?

Reply:
The functions in this file (`get_build_plan`, `run_mpyl`, `find_build_set` and `run_build`) are not (only) cli functions, so we specifically wanted to move them out of the `cli` folder. I admit `mpyl` might not be the best name, but these functions are the starting points of mpyl. I'm open for better suggestions!

Reply:
Can we just move it to src.mpyl?

Reply:
Actually we can't because then we end up with circular imports. `add_commands` imports `build` and build will import `find_build_set` again from `src.mpyl`

Reply:
What about `build.py`?